app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -86,20 +96,80 @@ def proof_with_threatmetrix_if_needed(
         end
       end
 
-      def proof_resolution(applicant_pii:, timer:)
+      def proof_residential_address_if_needed(
+        applicant_pii:,
+        timer:,
+        double_address_verification:
+      )
+        return residential_address_unnecessary_result unless double_address_verification
+
+        timer.time('residential address') do
+          resolution_proofer.proof(applicant_pii)
+        end
+      end
+
+      def residential_address_unnecessary_result
+        Proofing::AddressResult.new(
+          success: true, errors: {}, exception: nil, vendor_name: 'ResidentialAddressNotRequired',
```

Comment:
i'm unclear on the use of `vendor_name` here. The value of the hash seems more like a message about the proofing rather than about a specific vendor.

Reply:
I changed `vendor_name` to be `lexisnexis:instant_verify` and `aamva:state_id`, depending on the scenario, because that matches with what I've seen elsewhere in the codebase.  I'll try to follow up more on this, too.

Reply:
i think vendor name kind of ok, lexisnexis:instant_verify, it basically followings vendor:product pattern, if we need to include some other information, in this case, not required, we can leave it blank or use something like lexisnexis:instant_verify:no_op?